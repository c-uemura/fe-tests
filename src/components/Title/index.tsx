interface Props {
  title: string;
}
const Title = ({ title }: Props) => <h2>{title}</h2>;

export default Title;
