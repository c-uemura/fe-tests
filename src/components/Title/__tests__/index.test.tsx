import { render } from '@testing-library/react';
import Title from '@/components/Title';

describe('Title', () => {
  it('should display the title', () => {
    const title = 'Title Example';
    const { baseElement } = render(<Title title={title} />);
    expect(baseElement).toMatchSnapshot();
  });
});
