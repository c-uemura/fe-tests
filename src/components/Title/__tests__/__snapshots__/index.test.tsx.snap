// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Title > should display the title 1`] = `
<body>
  <div>
    <h2>
      Title Example
    </h2>
  </div>
</body>
`;
